### PR TITLE
[bugfix] cpp-test small bugs

### PR DESCRIPTION
**Pull request by PatriceJiang**

### CODE REVIEW

File: cocos/renderer/backend/opengl/CommandBufferGL.cpp [hunk 1]
```diff
@@ -522,6 +522,9 @@ void CommandBufferGL::setLineWidth(float lineWidth)
 {
     if(lineWidth > 0.0f)
         glLineWidth(lineWidth);
+    else
+        glLineWidth(1.0f);
+
```
Comment (minggo) [inline cocos/renderer/backend/opengl/CommandBufferGL.cpp:527]: it is strange, `1.f > 0.0f`, so will be set, why need to add this line

Comment (PatriceJiang) [reply cocos/renderer/backend/opengl/CommandBufferGL.cpp:527]: Default line width of some commands are `0`, that make `glLineWidth` remain unchanged if `setLineWidth` is not called, which state is wrong.

Comment (minggo) [reply cocos/renderer/backend/opengl/CommandBufferGL.cpp:527]: So this codes changed to
```c++
if (lineWidth >= 0.f)
    glLineWidth(lineWidth);
```
?

Comment (Mee-gu) [reply cocos/renderer/backend/opengl/CommandBufferGL.cpp:527]: The default width is 1:0 in OpengGL. Values less than or equal to 0:0 generate the
error INVALID_VALUE.

**Review by Mee-gu — APPROVED** [review]

File: cocos/renderer/backend/opengl/CommandBufferGL.cpp (hunk 1 above)
Comment (minggo) [reply cocos/renderer/backend/opengl/CommandBufferGL.cpp:527]: Then it is the error of our codes to pass 0?

**Pull request merged**